src/v2/view-builder/views/ov/EnrollPollOktaVerifyView.js

```diff
@@ -65,8 +65,7 @@ const Body = BaseFormWithPolling.extend(Object.assign(
         shouldStartPolling = true;
       } else if (['samedevice', 'devicebootstrap'].includes(selectedChannel)) { 
         // no selector if the channel is same device or device bootstrap
-        // additionally, stop polling as it should be a terminal page
-        shouldStartPolling = false;
+        shouldStartPolling = true;
```

Comment:
let't separate out same device and device bootstrap here. OV2OV is not shipping out with this together, so we should keep deviceBootstrap polling as false.
Also, let's make a follow up Jira to get test for it.

Reply:
follow up jira for test: https://oktainc.atlassian.net/browse/OKTA-798570